keyboards/etiennecollin/wave/readme.md

```diff
@@ -0,0 +1,27 @@
+# Wave
+
+![Wave pcb](https://i.imgur.com/oWF1Fnr.png)
```

Comment:
```suggestion
![Wave pcb](https://i.imgur.com/oWF1Fnrh.png)
```

Reply:
This breaks the transparency of my image... should I still use it?